src/eventbridge/send.ts

```diff
@@ -0,0 +1,41 @@
+import { EventBridge } from 'aws-sdk';
+import { EventEntry } from './EventEntry';
+import { Entries } from './Entries';
+import { SendResponse } from './SendResponse';
+import logger from '../observability/logger';
+import { MCRequest } from '../utils/MCRequest';
+
+const eventbridge = new EventBridge();
+// eslint-disable-next-line @typescript-eslint/require-await
+const sendMCProhibition = async (mcRequests: MCRequest[]): Promise<SendResponse> => {
+  const sendResponse: SendResponse = {
+    SuccessCount: 0,
+    FailCount: 0,
+  };
+  try {
+    for (let i = 0; i < mcRequests.length; i++) {
+      const x = mcRequests[i];
```

Comment:
You could potentially add a Joi.validate() here to validate the payload you're about to send.

You could also do a `for (const request in mcRequests)` to remove line 17

Reply:
I've tried adding that for loop in and i do agree, it's more readable to do it this way or using for of but ESLint doesn't like it being done in this way